app/models/concerns/account_interactions.rb

```diff
@@ -251,10 +251,13 @@ def lists_for_local_distribution
          .where('users.current_sign_in_at > ?', User::ACTIVE_DURATION.ago)
   end
 
-  def remote_followers_hash(url_prefix)
-    Rails.cache.fetch("followers_hash:#{id}:#{url_prefix}") do
+  def remote_followers_hash(url)
+    url_prefix = url[/\Ahttp(s?):\/\/[^\/]+/]
```

Comment:
It seems like the regex is a constant that is used in at least two places. Might be better to extract it to one.